env.dist

```diff
@@ -18,6 +18,18 @@ OAUTH_ORG_ID=
 OAUTH_SCOPES=AdobeID, openid, read_organizations, additional_info.projectedProductContext, additional_info.roles, adobeio_api, read_client_secret, manage_client_secrets, event_receiver_api, commerce.accs
 #OAUTH_HOST=<string>
 
+# These values can be copied from the Integration Details page in Commerce under System > Integrations
+# Commerce Namespaced Configs
+# The following values can be copied from the Credential details page in AppBuilder under Organization > Project > Workspace > OAuth Server-to-Server
+AIO_COMMERCE_IMS_CLIENT_ID=
```

Comment:
I may revert this addition depending on feedback. The goal of this namespace environment variable is to make it easier to developers and end users to understand that we're strictly talk about IMS usage within the Commerce domain and the OAUTH_XXX sounds very generic.